pkg/pxc/service.go

```diff
@@ -85,3 +86,28 @@ func NewServiceProxySQL(cr *api.PerconaXtraDBCluster) *corev1.Service {
 
 	return obj
 }
+
+func NewPodDistributedBudget(cr *api.PerconaXtraDBCluster, pdbSpec *policyv1beta1.PodDisruptionBudgetSpec, componentName string) *policyv1beta1.PodDisruptionBudget {
```

Comment:
I don't think that `service.go` is the right file for the `NewPodDistributedBudget` (just by the naming reasons)

Reply:
Should I make a new package for Pdb ?

Reply:
no, just move it to separate file or to the pxc.go. `service.go` contains code for generating `corev1.Service` so it's an unexpected place to look PDB for.